model/src/wav_import_export.F90

```diff
@@ -357,6 +359,11 @@ subroutine import_fields( gcomp, time0, timen, rc )
       if (state_fldchk(importState, 'So_u')) then
         call SetGlobalInput(importState, 'So_u', vm, global_data, rc)
         if (ChkErr(rc,__LINE__,u_FILE_u)) return
+        do isea = 1,nsea 
```

Comment:
This needs to be isolated so as to not to affect other users of the WW3 mesh cap.